src/platform/curses-platform.c

```diff
@@ -21,7 +21,48 @@ static void gameLoop() {
     Term.end();
 }
 
-static void curses_plotChar(uchar ch,
```

Comment:
See https://github.com/tmewett/BrogueCE/pull/82 for curses and web platform fixes